### PR TITLE
*: Speed up test 

### PR DESCRIPTION
**Pull request by zimulala**
The time spent before and after is "71s vs. 49s"
Reduce the DDL test failure because the scheme is out of date.

### CODE REVIEW

File: domain/domain.go [hunk 1]
```diff
@@ -492,11 +496,12 @@ func NewDomain(store kv.Storage, lease time.Duration) (d *Domain, err error) {
 	// Only when the store is local that the lease value is 0.
 	// If the store is local, it doesn't need loadSchemaInLoop and checkValidityInLoop.
 	if lease > 0 {
-		d.loadCh = make(chan time.Duration, 1)
 		d.checkCh = make(chan time.Duration, 1)
-		go d.loadSchemaInLoop(lease)
+		d.loadCh = make(chan time.Duration, 1)
 		go d.checkValidityInLoop(lease)
 	}
+	// Some tests need to reload schema when lease is 0.
```
Comment (shenli) [inline domain/domain.go]: Update this comment. This is not only used in test.

**Comment by zimulala** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**